docs/product-user-guide/src/main/asciidoc/chap-setting-up-a-new-project.adoc

```diff
@@ -406,13 +406,13 @@ A _KIE base_ is a repository of the application's knowledge definitions. It cont
 A _KIE session_ stores runtime data created from a KIE base. For more information, see the {URL_DEVELOPMENT_GUIDE}#sect_kie_sessions[KIE Sessions] chapter of the _{DEVELOPMENT_GUIDE}_.
 
 
-You can create KIE bases and sessions by editing the `kmodule.xml` project descriptor file of your project.You can do so through Business Central or by editing `kmodule.xml` in the `src/main/resources/META-INF/` folder by navigating through the `Repository` view.
+You can create KIE bases and sessions by editing the `kmodule.xml` project descriptor file of your project.You can do so through {CENTRAL} or by editing `kmodule.xml` in the `src/main/resources/META-INF/` folder by navigating through the `Repository` view.
```

Comment:
Missing space (see "...file of your project.You can...")

Reply:
@michelehaglund